tests/integration/psql_import/test_import_psql.py

```diff
@@ -219,15 +227,17 @@ def test_psql_import(
             )
             assert len(results[0].docs[0].matches) == 0
 
-            with TimeContext(f'### snapshotting {nr_docs} docs'):
-                flow_storage.post(
-                    on='/snapshot',
-                )
+            if snapshot:
+                with TimeContext(f'### snapshotting {nr_docs} docs'):
+                    flow_storage.post(
+                        on='/snapshot',
+                    )
 
             with TimeContext(f'### importing {nr_docs} docs'):
-                flow_query.post(
-                    on='/sync',
-                )
+                params = {}
+                if not snapshot:
+                    params = {'only_delta': True, 'startup': True}
+                flow_query.post(on='/sync', parameters=params)
```

Comment:
I think the following is better readable and easier to reasonable about:

```
            if snapshot:
                with TimeContext(f'### snapshotting {nr_docs} docs'):
                    flow_storage.post(
                        on='/snapshot',
                    )
                with TimeContext(f'### importing {nr_docs} docs'):
                    flow_query.post(on='/sync', parameters={})
            else:
                with TimeContext(f'### importing {nr_docs} docs'):
                    flow_query.post(on='/sync', parameters={'only_delta': True, 'startup': True})
```
Especially, since you want to showcase how to use snapshot and how not to.